pkg/server/server.go

```diff
@@ -433,3 +437,58 @@ func (s *Server) handleOpenShiftTokenDeletion(user *auth.User, w http.ResponseWr
 	io.Copy(w, resp.Body)
 	resp.Body.Close()
 }
+
+func (s *Server) handleHelmRenderManifests(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	var req helm.HelmRequest
+
+	err := json.NewDecoder(r.Body).Decode(&req)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	conf := actions.GetActionConfigurations(s.KubeAPIServerURL, req.Namespace, user.Token, &s.K8sClient.Transport)
+	resp, err := actions.RenderManifests(req.Name, req.ChartUrl, req.Values, conf)
+	if err != nil {
+		sendResponse(w, http.StatusBadGateway, apiError{fmt.Sprintf("Failed to render manifests: %v", err)})
+	}
+
+	w.Header().Set("Content-Type", "text/yaml")
+	w.Write([]byte(resp))
+}
+
+func (s *Server) handleHelmInstall(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	var req helm.HelmRequest
+
+	err := json.NewDecoder(r.Body).Decode(&req)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	conf := actions.GetActionConfigurations(s.KubeAPIServerURL, req.Namespace, user.Token, &s.K8sClient.Transport)
+	resp, err := actions.InstallChart(req.Namespace, req.Name, req.ChartUrl, req.Values, conf)
+	if err != nil {
+		sendResponse(w, http.StatusBadGateway, apiError{fmt.Sprintf("Failed to install helm chart: %v", err)})
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	res, _ := json.Marshal(resp)
+	w.Write(res)
+}
+
+func (s *Server) handleHelmList(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	params := r.URL.Query()
+	ns := params.Get("ns")
```

Comment:
please change the name of param to `namespace` to make it aligned with other two handlers.

Reply:
@pedjak this would require change in frontend as well

Reply:
UI changes are needed anyway since we have changed api paths here and moved us to json payload

Reply:
those are already merged https://github.com/openshift/console/pull/3850/files